vllm/model_executor/model_loader.py

```diff
@@ -58,11 +67,13 @@ def get_model(model_config: ModelConfig) -> nn.Module:
                 f"{supported_dtypes}")
         linear_method = quant_config.get_linear_method()
 
-    with _set_default_torch_dtype(model_config.dtype):
+    with _set_default_torch_dtype(
+            model_config.dtype), _set_default_torch_device(
+                model_config.device):
```

Comment:
To my knowledge, python 3.8 does not support the nested with statement in a single line. Could you make this into two separate lines (and add a note so that people don't accidentally change it back to a single line)?

Reply:
this change is unnecessary, revert back:)